bentoml/_internal/bento/build_config.py

```diff
@@ -592,7 +676,7 @@ def __attrs_post_init__(self) -> None:
                     if self.docker.cuda_version != "default" and (
                         self.docker.cuda_version not in _spec.supported_cuda_versions
                     ):
-                        raise BentoMLException(
+                        raise BentoMLException(  # pragma: no cover
```

Comment:
why no cover?

Reply:
Though I think this is an understandable no cover if we were aiming for 100% test coverage (obvious error cases probably don't need tests) it should probably be justified.